guidelines/index.html

```diff
@@ -1432,30 +1432,30 @@ <h1 id="title" class="title">Web Content Accessibility Guidelines (WCAG) 2.2</h1
-        <li>has a contrast ratio of at least 3:1 between the same pixels in the focused and unfocused states.</li>
+        <li>フォーカスされていないコンポーネント又はサブコンポーネントの厚さ 2 <a href="#dfn-css-pixels" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-css-pixels-3" title="約 0.0213 度の視野角。">CSS ピクセル</a>の<a href="#dfn-perimeter" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-perimeter-1" title="[New]">外周</a>の面積と少なくとも同じ大きさである。かつ、</li>
+        <li>同一ピクセルのフォーカス状態と非フォーカス状態との間で少なくとも 3:1 のコントラスト比がある。</li>
     </ul>
 
-    <p>Exceptions:</p>
+    <p>例外:</p>
     <ul>
-      <li>The focus indicator is determined by the <a href="#dfn-user-agents" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-user-agents-1" title="ウェブコンテンツを取得して利用者に提示するあらゆるソフトウェア。">user agent</a> and cannot be adjusted by the author, or</li>
-        <li>The focus indicator and the indicator's background color are not modified by the author.</li>
+      <li>フォーカスインジケータが<a href="#dfn-user-agents" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-user-agents-1" title="ウェブコンテンツを取得して利用者に提示するあらゆるソフトウェア。">ユーザエージェント</a>によって定められており、かつコンテンツ制作者による調整ができない場合。又は、</li>
+        <li>フォーカスインジケータ及びそのインジケータの背景の色を、コンテンツ制作者が変更していない場合。</li>
     </ul>
     
-    <div class="note" role="note" id="issue-container-generatedID-29"><div role="heading" class="note-title marker" id="h-note-29" aria-level="5"><span>注記 1</span></div><p class="">What is perceived as the user interface component or sub-component (to determine enclosure or size) depends on its visual <a href="#dfn-presentation" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-presentation-3" title="rendering of the content in a form to be perceived by users">presentation</a>. The visual presentation includes the component's visible <a href="#dfn-content" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-content-1" title="コンテンツの構造、提示、及びインタラクションを定義するコード又はマークアップも含めて、ユーザエージェントによって利用者に伝達される情報及び感覚的な体験。">content</a>, border, and component-specific background. It does not include shadow and glow effects outside the component's content, background, or border.</p></div>
+																																																																																													<div class="note" role="note" id="issue-container-generatedID-29"><div role="heading" class="note-title marker" id="h-note-29" aria-level="5"><span>注記 1</span></div><p class="">ユーザインタフェースコンポーネント又はサブコンポーネントとして認識されるもの (輪郭又は大きさを決定づけるもの) は、その視覚的<a href="#dfn-presentation" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-presentation-3" title="利用者が知覚できる形式でコンテンツをレンダリングすること。">提示</a>に依存する。視覚的提示には、コンポーネントの可視<a href="#dfn-content" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-content-1" title="コンテンツの構造、提示、及びインタラクションを定義するコード又はマークアップも含めて、ユーザエージェントによって利用者に伝達される情報及び感覚的な体験。">コンテンツ</a>、ボーダー、及びコンポーネント固有の背景が含まれる。コンポーネントのコンテンツ、背景、又はボーダーの外側にある、影及び光の効果は含まれない。</p></div>
```

Comment:
enclosureをどう訳すべきか迷いますね…
glow effectはグロー効果とするのはどうでしょうか。
```suggestion
    <div class="note" role="note" id="issue-container-generatedID-29"><div role="heading" class="note-title marker" id="h-note-29" aria-level="5"><span>注記 1</span></div><p class="">ユーザインタフェースコンポーネント又はサブコンポーネントとして認識されるもの (輪郭又は大きさを決定づけるもの) は、その視覚的<a href="#dfn-presentation" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-presentation-3" title="利用者が知覚できる形式でコンテンツをレンダリングすること。">提示</a>に依存する。視覚的提示には、コンポーネントの可視<a href="#dfn-content" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-content-1" title="コンテンツの構造、提示、及びインタラクションを定義するコード又はマークアップも含めて、ユーザエージェントによって利用者に伝達される情報及び感覚的な体験。">コンテンツ</a>、ボーダー、及びコンポーネント固有の背景が含まれる。コンポーネントのコンテンツ、背景、又はボーダーの外側にある、影及びグロー効果は含まれない。</p></div>
```

Reply:
enclosure ですが、コンポーネントの囲みを表すもの、と理解して、「輪郭」としてみました。素直に「囲み」とするほうがよいですかね...。

「グロー効果」は、よいと思います！

Reply:
以下の commit にて修正しました。
[379969b](https://github.com/waic/wcag22/pull/37/commits/379969bc1fd17c738efc8b13c8602e3aa2986475)

Reply:
> enclosure ですが、コンポーネントの囲みを表すもの、と理解して、「輪郭」としてみました。素直に「囲み」とするほうがよいですかね...。

（CSSでいうところの）outlineのことをいっている（？）とするなら、「輪郭」でよい気がしてきました。

Reply:
そうですね。私もそんな気がするので、やはり「輪郭」に戻そうと思います。